app/templates/application.hbs

```diff
@@ -14,7 +14,7 @@
         {{if layoutManager.leftSideBarOpen "left-sidebar-open"}}'
     >
       <div class='row justify-content-center'>
-        <div class='col-12 col-md-8 col-lg-9 col-xl-10'>
+        <div class='col-12 col-md-10 col-lg-9 col-xl-10'>
```

Comment:
Are you sure this doesn't affect the layout of any other pages?

Reply:
In my head I thought it was weird that it went like: 12 8 9 like md randomly is smaller, so I changed it to be 12 10 9, so as the screen gets bigger it takes up less percent of the space. But I didnt see xl-10, and with that observation my change does not make any sense. So Im changing it back, better be safe than sorry.